pygmt/helpers/decorators.py

```diff
@@ -375,10 +375,16 @@ def new_module(*args, **kwargs):
             for arg, alias in aliases.items():
```

Comment:
I find the variables 'arg' and 'alias' confusing. Maybe we should be more explicit and call them 'short_param' (e.g. -J) and 'long_alias' (e.g. projection) instead? What do you think?

```suggestion
            for short_param, long_alias in aliases.items():
```

Reply:
Done in 30cf3e2cd2f7e33ad2fbc37cf081fdc02442899a.

Reply:
Looks much better, thanks!